docs/getting-started/other-notes.md

```diff
@@ -0,0 +1,72 @@
+# Other Notes
+
+
+```{warning}
+name under devlopment:\
+    Change Log\
+    Other Notes\
+    Tweak Notes\
+    Improvement Changes\
+    Specification improvements\
+```
+
+For release notes go to [Release Notes](release-notes.md). All the other (smaller) changes are documented here. This includes especially changes in wording and visuals. Improvements in the specification are not changing the functionality or compatibility of SAF. These changes only improve the understandability of SAF documentation.
+
+
+## 9.12.2021 Transfer to Read the Docs and other changes
+In order to move to new provider [Read the Docs](https://readthedocs.org/), it was nescesary do extensive changes in syntax. Hints and notes visuals were changed, images sizes were set and links were updated. No content was edited.
+
+[Other Notes](other-notes.md)
+* New page Other Notes was created to separate smaller changes in documentation from [Release Notes](release-notes.md)
+
+[StructuralSurfaceActionDistribution](../loads/structuralsurfaceactiondistribution-1.md)
+* Object renamed to StructuralSurfaceActionDistribution to keep the same naming across the documentation. The only place where the name is shorter is in excel sheet name, where the full name doesn't fit.
+
+[Ignored Objects and Groups](project-and-model-specifications/ignore.md)
+* Updated to contain all the objects
+* StructuralProxyElement, StructuralStorey, StructuralCurveMemberVarying, StructuralCurveMemberVarying and StructuralSurfaceActionDistribution were added
```

Comment:
I would include the main lines in the bulleted list too:

* [Other Notes](other-notes.md)
  * New page Other Notes was created to separate smaller changes in documentation from [Release Notes](release-notes.md)
* [StructuralSurfaceActionDistribution](../loads/structuralsurfaceactiondistribution-1.md)
  * Object renamed to StructuralSurfaceActionDistribution to keep the same naming across the documentation. The only place where the name is shorter is in excel sheet name, where the full name doesn't fit.
* [Ignored Objects and Groups](project-and-model-specifications/ignore.md)
  * Updated to contain all the objects
  * StructuralProxyElement, StructuralStorey, StructuralCurveMemberVarying, StructuralCurveMemberVarying and StructuralSurfaceActionDistribution were added

Reply:
The above applies for every list on this page btw

Reply:
And maybe apply this to the release notes page too? It'll cut down on the amount of whitespace dramatically

Reply:
I changed the structure for change log and release notes as well, I wasn't sure if it isn't too squashed. So I tried putting in blank spaces for a test in the end of modified objects under 18.11.2021. Which one do you think looks the best?

Reply:
Nah, your "modified objects" and "new objects added" sub-titles already act as spacers for your blocks.
It is what is is, you're "listing" your changes so it makes sense that it's, well, a list 😄